zilencer/management/commands/populate_db.py

```diff
@@ -755,9 +755,14 @@ def handle(self, **options: Any) -> None:
                 # suite fast, don't add these users and subscriptions
                 # when running populate_db for the test suite
 
+                # to imitate emoji insertions in stream names
+                raw_emojis = ["😎", "😂", "🐱‍👤"]
```

Comment:
I think this array is probably a little silly, and we should just put the emoji inline below.

Reply:
the reason I put these emojis in an array is because this array will also be used in defining extra-stream names `populate_db line 815`. this line currently works only if --extra-users > 300 argument is also passed as I have defined the existing raw_emojis array there. 

this new array will allow us to pass --extra-streams argument without --extra-users as it should be